.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,3 +28,27 @@ I have:
 
 [1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a
 [2]: https://www.notion.so/passcultureapp/cb45383351b44723a6f2d9e1481ad6bb?v=10fe47258701423985aa7d25bb04cfee&pvs=4
+
+## Best Practices
+
+<details>
+  <summary>Click to expand</summary>
+
+- Don't use type assertions with `as` (type assertions are removed at compile-time, there is no runtime checking associated with a type assertion. There won’t be an exception or `null` generated if the type assertion is wrong).
+- Don't opt out of type checking with `any` (when you want to accept anything because you will be blindly passing it through without interacting with it, you can use `unknown`).
```

Comment:
je suis nul en anglais "don't opt out of type checking", je ne comprends pas (après ayant marqué la phrase en français, j'imagine ce que ça veut dire)